js/form_editor_controller.js

```diff
@@ -389,6 +389,13 @@ class GlpiFormEditorController
             } else if (type === "question") {
                 // The input is for a question
                 base_input_index =  `_questions[${item_index}]`;
+
+                // Check if the input is an option (has the data-glpi-form-editor-specific-question-extra-data attribute)
+                let is_option = $(input).attr("data-glpi-form-editor-specific-question-extra-data") !== undefined;
```

Comment:
```suggestion
                const is_option = $(input).attr("data-glpi-form-editor-specific-question-extra-data") !== undefined;
```

If the value is never redefined, it should be a `const`.